### ISSUE TITLE
User settings page input exist email alert message error

### ISSUE THREAD

**Issue by yzijun** [jhipster/generator-jhipster#2562]
JHipster Version

2.25.0
Reproduce the error

url:http://localhost:3000/#/settings
Page file: scripts\app\account\settings\settings.html

When i input email user@localhost alert message  An error has occurred! Settings could not be saved.
login user is admin

Suggest a Fix
js file:scripts\app\account\settings\settings.controller.js
```
angular.module('sampleApplicationApp')
    .controller('SettingsController', function ($scope, Principal, Auth, Language, $translate) {
        $scope.success = null;
        $scope.error = null;
        Principal.identity(true).then(function(account) {
            $scope.settingsAccount = account;
        });

        $scope.save = function () {
            Auth.updateAccount($scope.settingsAccount).then(function() {
                $scope.error = null;
                $scope.success = 'OK';
                Principal.identity().then(function(account) {
                    $scope.settingsAccount = account;
                });
                Language.getCurrent().then(function(current) {
                    if ($scope.settingsAccount.langKey !== current) {
                        $translate.use($scope.settingsAccount.langKey);
                    }
                });
            }).catch(function() {
                $scope.success = null;
                $scope.error = 'ERROR';
            });
        };
```
modified
```
angular.module('sampleApplicationApp')
    .controller('SettingsController', function ($scope, Principal, Auth, Language, $translate) {
        $scope.success = null;
        $scope.error = null;
        $scope.errorEmailExists = null;
        Principal.identity(true).then(function(account) {
            $scope.settingsAccount = account;
        });

        $scope.save = function () {
            Auth.updateAccount($scope.settingsAccount).then(function() {
                $scope.error = null;
                $scope.errorEmailExists = null;
                $scope.success = 'OK';
                Principal.identity().then(function(account) {
                    $scope.settingsAccount = account;
                });
                Language.getCurrent().then(function(current) {
                    if ($scope.settingsAccount.langKey !== current) {
                        $translate.use($scope.settingsAccount.langKey);
                    }
                });
            }).catch(function(httpResponse) {
                $scope.success = null;
                var errorHeader = httpResponse.headers('X-sampleApplicationApp-error');
                if (errorHeader === 'error.emailexists') {
                    $scope.errorEmailExists = 'ERROR';
                } else {
                    $scope.error = 'ERROR';
                }
            });
        };
```
I'm sorry
My English so poor

**Comment by deepu105** [comment]
can you propose a PR?

**Comment by yzijun** [comment]
I'm not sure.
I can try.

**Comment by rkohel** [comment]
This issue should be fixed by #2460 already.
Please try again with 2.26.2.

**Comment by yzijun** [comment]
@rkohel thank you
